### DISCUSSION TITLE
Vídeo 09 - Desenvolvimento

### DISCUSSION THREAD

**Discussion by thpoiani** [thpoiani/lab-quarkus#12] [General]
Estamos usando um Scheduler/cron para uma tarefa periódica de sincronismo entre o Redis e MariaDB.

Quais outras soluções você consegue pensar para substituir esse cron?

**Comment by didifive** [comment]
Uma alternativa sem scheduler/cron: utilizar Pub/Sub do Redis para que o voting-app publique que recebeu voto e a election-manager subscrita já executa o sync. Pode ser também utilizando um gerenciador de fila externo.
Fora isso vou acompanhar para ver outras maneiras propostas
